gallery/src/pages/Home.tsx

```diff
@@ -20,34 +20,32 @@ const Home = () =>
     <p className="nx-p">
       The sidebar at left can be used to navigate to the various examples, which are organized into groups as follows:
     </p>
-    <dl className="nx-list nx-list--definition-list">
-      <dt className="nx-list__item nx-list__item--label">React Components</dt>
-      <dd className="nx-list__item">
-        Documentation of all React components available to consumers of RSC, including demonstrations, example
-        code, and full documentation of relevant React <code className="nx-code">props</code>. Examples here
-        demonstrate basic usage of each component along with some modicum of supporting code - for example,
-        code tracking the toggle state of a <code className="nx-code">NxCheckbox</code>. This supporting code
-        should be taken as example only. Depending on the architectural needs of the consuming application, there
-        are many different ways that state management and other supporting concerns could be handled.
-      </dd>
-      <dt className="nx-list__item nx-list__item--label">Guidelines</dt>
-      <dd className="nx-list__item">Miscellaneous advice for RSC consumers</dd>
-      <dt className="nx-list__item nx-list__item--label">Styles - HTML Elements</dt>
-      <dd className="nx-list__item">
-        Demonstrations of raw CSS classes that are available to consumers of RSC. Some of these classes are also
-        available as React components, but are documented here as well for lower-level/non-React use.
-      </dd>
-      <dt className="nx-list__item nx-list__item--label">Styles - Mixins &amp; Helpers</dt>
-      <dd className="nx-list__item">
-        Documentation of SCSS mixins and CSS helper classes available to consumers of RSC.
-      </dd>
-      <dt className="nx-list__item nx-list__item--label">Layout Examples</dt>
-      <dd className="nx-list__item">
-        Explanations of broader layout concepts that includes groups of components and/or styles working together.
-      </dd>
-      <dt className="nx-list__item nx-list__item--label">JavaScript &amp; TypeScript Utilities</dt>
-      <dd className="nx-list__item">Utility function and datatypes available to users of RSC.</dd>
-    </dl>
+    <h3 className="nx-h3">React Components</h3>
+    <p className="nx-p">
+      Documentation of all React components available to consumers of RSC, including demonstrations, example
+      code, and full documentation of relevant React <code className="nx-code">props</code>. Examples here
+      demonstrate basic usage of each component along with some modicum of supporting code - for example,
+      code tracking the toggle state of a <code className="nx-code">NxCheckbox</code>. This supporting code
+      should be taken as example only. Depending on the architectural needs of the consuming application, there
+      are many different ways that state management and other supporting concerns could be handled.
+    </p>
+    <h3 className="nx-h3">Guidelines</h3>
```

Comment:
Should these be `h4`s? It's weird that the subsection headings that follow this part have headers with the same style and of the same type, even though they represent a larger separation

Reply:
> Should these be `h4`s? It's weird that the subsection headings that follow this part have headers with the same style and of the same type, even though they represent a larger separation

I'm not sure that I understand, the text above reads "which are organized into groups as follows:" and then the groups are described each with an `H3` (which is correct) and corresponding to the sidebar navigation groups, if some of the groups were `H4`'s which would imply they were sub-groups of one of the groups which is not the case

Reply:
> I'm not sure that I understand, the text above reads "which are organized into groups as follows:" and then the groups are described each with an H3 (which is correct) and corresponding to the sidebar navigation groups, if some of the groups were H4's which would imply they were sub-groups of one of the groups which is not the case

The problem is that "JavaScript & TypeScript Utilities" and "Consuming the React Shared Components" are both `h3`s even though they are not conceptually on the same level, and even though "Consuming the React Shared Components" is a tile subsection while "JavaScript & TypeScript Utilities" is not.  Maybe what we really need is a subsection + subsection header for the whole part that's about the navigation ("Navigating the Gallery" or something) and then "JavaScript & TypeScript Utilities" et. al. would be `h4`s within that subsection.

Reply:
> > I'm not sure that I understand, the text above reads "which are organized into groups as follows:" and then the groups are described each with an H3 (which is correct) and corresponding to the sidebar navigation groups, if some of the groups were H4's which would imply they were sub-groups of one of the groups which is not the case
> 
> The problem is that "JavaScript & TypeScript Utilities" and "Consuming the React Shared Components" are both `h3`s even though they are not conceptually on the same level, and even though "Consuming the React Shared Components" is a tile subsection while "JavaScript & TypeScript Utilities" is not. Maybe what we really need is a subsection + subsection header for the whole part that's about the navigation ("Navigating the Gallery" or something) and then "JavaScript & TypeScript Utilities" et. al. would be `h4`s within that subsection.

This is way too far into the weeds for this ticket, I think it's fine as it is and if @dturner32 wants it tweaked then he can provide direction. To me the problem isn't with the `h3`'s that were changed, the problem is with the sub-section styling below which isn't a part of this ticket.

I'm not opposed to converting the two sub-section headings to `h2`'s.

Reply:
> I'm not opposed to converting the two sub-section headings to h2's.

Subsection headings must be h3s, as the tile header is an h2.  I'll fix this after you merge to the feature branch if you don't want to do it here.

Reply:
> > I'm not opposed to converting the two sub-section headings to h2's.
> 
> Subsection headings must be h3s, as the tile header is an h2. I'll fix this after you merge to the feature branch if you don't want to do it here.

If you want, as long as there is a PR